optimum/intel/openvino/modeling_base.py

```diff
@@ -346,6 +346,11 @@ def compile(self):
                 ov_config["CACHE_DIR"] = str(cache_dir)
                 logger.info(f"Setting OpenVINO CACHE_DIR to {str(cache_dir)}")
             self.request = core.compile_model(self.model, self._device, ov_config)
+            # OPENVINO_LOG_LEVEL can be found in https://docs.openvino.ai/2023.2/openvino_docs_OV_UG_supported_plugins_AUTO_debugging.html
```

Comment:
ov::log::level, check also this value, environment variable OPENVINO_LOG_LEVEL has higher priority.

Reply:
CPU and GPU plugin do not support LOG_LEVEL
RuntimeError: Exception from src/inference/src/core.cpp:113:
[ GENERAL_ERROR ] Exception from src/plugins/intel_cpu/src/config.cpp:277:
NotFound: Unsupported property LOG_LEVEL by CPU plugin.

Reply:
Only AUTO supports LOG_LEVEL.
e.g. For llm_bench  -d AUTO:CPU -lc config.txt,  ({"LOG_LEVEL":"LOG_INFO"} in config.txt)
AUTO:CPU SUPPORTED_PROPERTIES:
  NETWORK_NAME: Model0
  EXECUTION_DEVICES: ['CPU']
  PERFORMANCE_HINT: PerformanceMode.LATENCY
  OPTIMAL_NUMBER_OF_INFER_REQUESTS: 1
  MULTI_DEVICE_PRIORITIES: CPU
  CPU:
    AFFINITY: Affinity.CORE
    CPU_DENORMALS_OPTIMIZATION: False
    CPU_SPARSE_WEIGHTS_DECOMPRESSION_RATE: 1.0
    ENABLE_CPU_PINNING: True
    ENABLE_HYPER_THREADING: False
    EXECUTION_DEVICES: ['CPU']
    EXECUTION_MODE_HINT: ExecutionMode.PERFORMANCE
    INFERENCE_NUM_THREADS: 96
    INFERENCE_PRECISION_HINT: <Type: 'bfloat16'>
    NETWORK_NAME: Model0
    NUM_STREAMS: 1
    OPTIMAL_NUMBER_OF_INFER_REQUESTS: 1
    PERFORMANCE_HINT: LATENCY
    PERFORMANCE_HINT_NUM_REQUESTS: 0
    PERF_COUNT: NO
    SCHEDULING_CORE_TYPE: SchedulingCoreType.ANY_CORE
  MODEL_PRIORITY: Priority.MEDIUM
  LOADED_FROM_CACHE: False